src/enterprise.ts

```diff
@@ -20,22 +22,33 @@ export function onGitHubEnterprise (): boolean {
 }
 
 export async function getEnterpriseConfigUrl (configType: string, apiURL: string, version: string, token: string, owner: string, repository: string): Promise<string> {
-  let assetname: string = ''
-  let filename: string = ''
+  debug('Getting enterprise config URL')
+  if (configType !== DEFAULT_CONFIG && configType !== STAGE_CONFIG) return ''
 
-  if (configType === DEFAULT_CONFIG) {
-    assetname = ENTERPRISE_DEFAULTS_FILENAME_ON_RELEASE
-    filename = ENTERPRISE_DEFAULTS_FILENAME
-  } else if (configType === STAGE_CONFIG) {
-    assetname = ENTERPRISE_STAGE_CONFIG_FILENAME
+  debug('initiating assetName and filename')
+  let assetName: string = ENTERPRISE_DEFAULTS_FILENAME_ON_RELEASE
+  let filename: string = ENTERPRISE_DEFAULTS_FILENAME
+
+  if (configType === STAGE_CONFIG) {
+    debug('configType is STAGE_CONFIG')
+    assetName = ENTERPRISE_STAGE_CONFIG_FILENAME
     filename = ENTERPRISE_STAGE_CONFIG_FILENAME
-  } else {
-    return ''
   }
 
+  // if version starts with devel: then it should use inner source Piper
+  if (version.startsWith('devel:')) {
+    debug(`version starts with "devel:" => ${version}`)
+    debug(`params: ${owner}, ${repository}, ${version}, ${filename}`)
+    listFilesAndFolders(process.cwd())
+    version = ''
```

Comment:
I suppose it should be `latest` here as well?

Reply:
either way its gets turned to latest in the getTag function but I changed it